hubapi/deployment.py

```diff
@@ -23,7 +23,9 @@ def read_environment():
     password = str_to_ascii_to_base64_to_str(os.environ.get('JINA_DB_PASSWORD'))
     database_name = str_to_ascii_to_base64_to_str(os.environ.get('JINA_DB_NAME'))
     collection_name = str_to_ascii_to_base64_to_str(os.environ.get('JINA_DB_COLLECTION'))
-    return hostname, username, password, database_name, collection_name
+    docker_username = str_to_ascii_to_base64_to_str(os.environ.get('JINA_DOCKER_USERNAME'))
```

Comment:
Make sure these env variables are stored as secrets in Github.

Reply:
Done